pxtcompiler/emitter/service.ts

```diff
@@ -851,7 +857,7 @@ namespace ts.pxtc.service {
 
     const operations: ServiceOps = {
         reset: () => {
-            service.cleanupSemanticCache();
+            service = ts.createLanguageService(host)
```

Comment:
Can hide this behind a param to reset if it's concerning, but cleanupSemanticCache wasn't doing enough to clear / still resulting growing significantly and slowing things down / using tons of memory.

Reply:
as long as there isn't any noticeable perf drop, i think it's fine